pkg/model/types_test.go

```diff
@@ -9,6 +9,7 @@ import (
 
 var instId = "my-instance"
 var ip = "192.168.0.1"
+var serviceType = "ClusterSetIP"
```

Comment:
can we use the const?

Reply:
Tried, says `import cycle not allowed in test`, so I'm assuming you're not allowing to import the models.

Reply:
I believe that's if you try to use test-constants since that imports the model pkg.
I meant the const in types.go, which you should already have access to since the test is in same package.

Reply:
I see, changed to the const.